docs/5.0/preview/cloud.md

```diff
@@ -0,0 +1,150 @@
+To request access age get on the waiting list [here](https://goteleport.com/get-started/).
+
+## Status
+
+Teleport Cloud is currently only recommended for environments that do not require
+strict compliance standards or SLA uptime guarantees.
+
+We have designed Teleport Cloud environment to be secure; however we are still in the process of
+scrutinizing and executing on our security roadmap and working with independent security
+auditors to identify any gaps. Only once this is complete will we be able to to verify 
+that it is ready for strict compliance use-cases.
+
+## Roadmap
+
+For the first half of 2021, we will be working with auditors to realize the
+threat modeling and security controls.
+
+Once we realize, test and audit the security model we will start working towards meeting
+compliance programs such as FedRAMP.
+
+## Billing
+
+Usage data will be collected, but billing will not be set up until February.
+The preview version of Teleport Cloud is provided with free of charge primarily
+for testing and POC purposes until we commit to a formal SLA. 
+
+## Managed Teleport Settings
+
+SSH sessions are recorded [on nodes](../architecture/nodes.md#session-recording).
+Teleport Cloud Proxy does not terminate SSH sessions when using OpenSSH and `tsh` sessions.
+It terminates Application, Database and Kubernetes sessions on the proxy.
+
+## Data retention
+
+The session records are stored in S3 using at-rest encryption.
+We have yet to define specific retention policies.
+
+Customer data, including audit logging, is backed up using the DynamoDB
+"point in time recovery" system. Data can be recovered up to 35 days.
+This retention period is not configurable.
+
+## High availability
+
+Clusters are deployed in a single AWS region in 2 availability zones.
+AWS guarantees [99.99%](https://aws.amazon.com/compute/sla/) of monthly uptime percentage.
+
+Teleport Cloud preview version does not commit to any SLA. It sets availability
+objective to 99.7% of monthly uptime percentage (a maximum of two hours of downtime per month).
+
+We plan to get to 99.99% uptime SLA.
+
+# FAQ
+
+### Can I use Teleport Cloud Preview in production?
+
+We expect Teleport Cloud to be OK to use in non mission critical systems
+that do not require 24/7 access or a guaranteed SLA. We recommend a 
+fallback option be make available. 
+
+### Are you using AWS-managed encryption keys, or CMKs via KMS?
+
+We use AWS-managed keys. Currently there is no option to provide your own key.
+
+### Is this Teleport's S3 bucket, or my bucket based on my AWS credentials?
+
+It's a Teleport-managed S3 bucket with AWS-managed keys.
+Currently there is no way to provide your own bucket.
+
+### How do I add nodes to Teleport Cloud?
+
+You can connect servers, kubernetes clusters, databases and applications
+using [reverse tunnels](../admin-guide.md#adding-a-node-located-behind-nat).
+
+There is no need to open any ports on your infrastructure for inbound traffic.
```

Comment:
What's the outbound requirements for strict network requirements, should we also advise any IPs/FQDN to whitelist?

Reply:
We don't promise to keep any IPs, I don't think AWS promises anything either

Reply:
They can whitelist [these](https://ip-ranges.amazonaws.com/ip-ranges.json) if they like :)